extensions/prototypes/labgraph_monitor/src/components/SettingPanel/SettingPanel.tsx

```diff
@@ -106,6 +149,20 @@ const SettingPanel: React.FC = (): JSX.Element => {
                         <ChevronRight />
                     </IconButton>
                 </Box>
+
+                {mode === 'light' ? (
+                    <div
+                        id="dragger"
+                        onMouseDown={(e) => handleMouseDown(e)}
+                        className={classes.dragger_light}
+                    />
+                ) : (
+                    <div
+                        id="dragger"
+                        onMouseDown={(e) => handleMouseDown(e)}
+                        className={classes.dragger_dark}
+                    />
+                )}
```

Comment:
`Compiled with problems:X

ERROR in src/components/SettingPanel/SettingPanel.tsx:156:32

TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'ClassNameMap<"root" | "settingButton" | "settingIcon" | "settingPanel" | "themeBar" | "dragger_light" | "dragger_dark">'.
  No index signature with a parameter of type 'string' was found on type 'ClassNameMap<"root" | "settingButton" | "settingIcon" | "settingPanel" | "themeBar" | "dragger_light" | "dragger_dark">'.
    154 |                     id="dragger"
    155 |                     onMouseDown={handleMouseDown}
> 156 |                     className={classes['dragger_' + mode]}
        |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
    157 |                 />`
    
    I get this error when I try the above suggestion

Reply:
Used this suggestion instead
`className={mode === 'light' ? classes.dragger_light : classes.dragger_dark}`